lectureDL.py

```diff
@@ -125,10 +126,24 @@
 FOLDER_NAME_ERROR = ("There is a name mismatch between the subjects list and" +
                      " the folder names.\nYou might want to try the " +
                      "'auto_create_subfolders' option in the settings.")
+GET_ECHO = 'Getting past intermediate page / waiting for Echocenter to load...'
+FIRST_CHAR = 0
```

Comment:
I feel like this isn't particularly necessary, the meaning `0` seems very easy to infer from the context.

Reply:
Just improves human readability, IMO. Easy to infer, sure, but just saves the half-second inference process that builds up when looking over so many lines of code. Regardless, if you'd like, I'm happy to take your lead on this.